aicsimageio/vendor/omexml.py

```diff
@@ -35,16 +34,16 @@ def xsd_now():
 #
 # The namespaces
 #
-NS_BINARY_FILE = "http://www.openmicroscopy.org/Schemas/BinaryFile/2013-06"
+NS_BINARY_FILE = "http://www.openmicroscopy.org/Schemas/BinaryFile/2016-06"
 NS_ORIGINAL_METADATA = "openmicroscopy.org/OriginalMetadata"
-NS_DEFAULT = "http://www.openmicroscopy.org/Schemas/{ns_key}/2013-06"
+NS_DEFAULT = "http://www.openmicroscopy.org/Schemas/{ns_key}/2016-06"
```

Comment:
is it worth forcing ns_key to uppercase? Or is that done elsewhere?

Reply:
Because none of us have really ever touched this object (with the exception of Madison iirc), I don't want to change this too much in the PR. This is the minimum changes required to fix the issue linked and I am personally happy with that.